src/helper/service-worker-init.ts

```diff
@@ -0,0 +1,36 @@
+export function registerServiceWorker() {
+    if ('serviceWorker' in navigator) {
+        const isServiceWorkerEnabled = import.meta.env.VITE_SERVICE_WORKER_ENABLED === 'true';
+
+        if (isServiceWorkerEnabled) {
+            navigator.serviceWorker
+                .register('/service-worker.js')
+                .then((registration) => {
+                    console.log('Service Worker registered with scope:', registration.scope);
```

Comment:
These console logs don't serve any purpose for the user and thus should be removed.
This would also elevate the need for mocking the console API during unit testing.